app/views/admin/system_announcements/_system_announcement.html.slim

```diff
@@ -0,0 +1,26 @@
+hr
+div.announcement class=(system_announcement.valid_in_future? ? '' : 'future')
```

Comment:
this might be a helper, so it looks like
```slim
div.announcement class=future_class(system_announcement)
```
together with the module to augment `time_bounded`.